setupTests.js

```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line
+import raf from 'tempPolyfills'
```

Comment:
What is for raf used?

Reply:
it's need for "requestAnimationFrame warning with React 16"